orm/internal/testpb/bank.cosmos_orm.go

```diff
@@ -37,41 +37,41 @@ func (i BalanceIterator) Value() (*Balance, error) {
 
 type BalanceIndexKey interface {
 	id() uint32
-	values() []interface{}
+	values() []any
```

Comment:
Have you manually changed this files are generated it? If manually, please revert.
By the way, we should exclude generated files from the linter checks.

Reply:
will do both, beginning with a PR for generated files